From aebd475c359c94f5d6444e7f12a28dc552c1e634 Mon Sep 17 00:00:00 2001
From: Jiahua Wang <jiahwang@cisco.com>
Date: Fri, 3 Mar 2017 18:03:03 -0800
Subject: [PATCH] Add kexec reboot string

Signed-off-by: Jiahua Wang <jiahwang@cisco.com>
---
 kernel/kexec.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/kexec.c b/kernel/kexec.c
index 4b8f0c9..52cf984 100644
--- a/kernel/kexec.c
+++ b/kernel/kexec.c
@@ -1681,7 +1681,7 @@ int kernel_kexec(void)
 #endif
 	{
 		kexec_in_progress = true;
-		kernel_restart_prepare(NULL);
+		kernel_restart_prepare("kexec reboot");
 		migrate_to_reboot_cpu();
 
 		/*
-- 
2.1.4

